permission-element.bs

```diff
@@ -0,0 +1,25 @@
+<pre class="metadata">
+Title: The HTML &lt;permission&gt; Element
+Status: CG-DRAFT
+Group: WICG
+URL: https://github.com/WICG/pepc
```

Comment:
I'd suggest renaming this file to `index.bs` which would make the final URL here `https://wicg.github.io/PEPC/` rather than `https://wicg.github.io/PEPC/permission-element.html`. Either way this field should change to the URL of the spec document rather than the repository.

Reply:
My thinking was to not make it `index.html`, until the draft has some substance.